_posts/2022-11-17-Multiple-Authentication.md

```diff
@@ -0,0 +1,51 @@
+5. Simplify troubleshooting external IDP related issues
+
+## Use cases
+
+Customers using dedicated third-party enterprise-based IDPs to manage OpenSearch Dashboards user identities can now use OpenSearch as an Internal IDP for admin, security, and privileged accounts, and a single enterprise-based external IDP to maintain authentication information for general Dashboards user accounts. For larger enterprises that may have redundancy needs, they can enable multiple enterprise-based IDPs, for example, an enterprise OIDC IDP, and an enterprise SAML IDP.
+
+An additional use case for this that hasn’t been possible before is to add multiple external IDPs to mix and match needs. For example, OpenSearch can serve as an Internal IDP to maintain authentication information for admin, and security accounts, an Enterprise based IDP for privileged and analyst accounts, and a Social-based IDP for limited read-only accounts that can only view specific data.
+
+We would love to hear any additional use-cases you have that this feature can meet!
+
+## How can you use this feature?
+
+OpenSearch Dashboards admins can enable single or multiple authentication options on demand and customize the integrated Login UI by editing opensearch_dashboards.yml. For the essential settings required to setup multiple authentication, see [**Multiple Authentication Options**]({{site.url}}{{site.baseurl}}/security-plugin/configuration/multi-auth/). Once these options are configured, the Login UI will be automatically updated. For example, the login UI below shows both basic authentication and SSO enabled:
+
+![Default Login UI](/assets/media/blog-images/2022-11-14-multiple-authentication/default-login-ui.png){: .img-fluid}
+
+## How can you use this feature?
+
+When Dashboards bootstraps and loads security settings from `opensearch-dashboards.yml`, the Dashboards client can tell whether Single-Authentication Mode or Multiple-Authentication Mode is enabled by evaluating the Multiple Authentication feature flag and authentication type setting.
+
+* If **Single-Authentication Mode** is detected, only authentication handler for corresponding authentication type defined by *`attribute:opensearch_security.auth.type`* will be registered with the client. All the login requests flow to the dedicated IDP to complete identification and authentication.
+* If **Multiple-Authentication Mode** is detected, OSD will evaluate authentication handlers required for each individual authentication type defined by *`attribute:opensearch_security.auth.type`* and form a compound authentication handler, which will be registered with OSD client instead. This compound authentication handler is named as multi-auth handler. Multi-auth handler is able to redirect the authentication request to various IDP endpoints for authentication interchangeably based on the authentication type defined within Login request. 
+
+**High Level Diagram:**
```

Comment:
```suggestion
**High-level diagram**
```

Reply:
Rather than use a heading to introduce this image, I suggest we introduce it with a sentence that says something like "The following image shows..."

Reply:
Changed it to `The following image shows the high-level diagram for Multiple Authentication Options for OpenSearch Dashboards:`

Reply:
> Changed it to `The following image shows the high-level diagram for Multiple Authentication Options for OpenSearch Dashboards:`

`The following image shows a high-level diagram of the multiple authentication options for OpenSearch Dashboards.`